src/puppetlabs/dujour/version_check.clj

```diff
@@ -0,0 +1,116 @@
+(ns puppetlabs.dujour.version-check
+  (:require [clojure.tools.logging :as log]
+            [schema.core :as schema]
+            [ring.util.codec :as ring-codec]
+            [puppetlabs.http.client.sync :as client]
+            [cheshire.core :as json]
+            [trptcolin.versioneer.core :as version]
+            [slingshot.slingshot :as sling]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Constants
+
+(def default-group-id "puppetlabs.packages")
+(def default-update-server-url "http://updates.puppetlabs.com")
```

Comment:
I'm not sure if the server is configured to accept https conns or not; all of the other apps that I looked at are using http.  We can address that in a separate ticket if needed.